test/unit/rh_cloud_http_proxy_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_plugin_helper'
+
+class RhCloudHttpProxyTest < ActiveSupport::TestCase
+  setup do
+  end
+
+  test 'Proxy selected in the correct order' do
+    global_content_proxy_mock = 'http://global:content@localhost:8888'
+    katello_cdn_proxy_mock = 'http://katello:cdn@localhost:8888'
```

Comment:
cdn setting is a hash, so the value must be a hash too:

```suggestion
    katello_cdn_proxy_mock = {
      host: 'localhost',
      port: '8888',
      user: 'katello',
      password: 'c@n',
      scheme: 'http'
    }
```